Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/UserMenuController.java

```diff
@@ -213,17 +213,19 @@ public void onEvent(Event event) {
                         try {
                             boolean logoutSuccess = false;
 
+                            final Session session = Sessions.getCurrent();
+
                             if (config.isUseKeycloakSso()) {
-                                logoutSuccess =
-                                        managerService.logoutUserAllSessions(
-                                                currentUser.getUsername(),
-                                                config.getSecurityMsHttpLogoutUrl(),
-                                                config.getSecurityMsHttpsLogoutUrl());
-                                LOGGER.info("\nlogoutSuccess: {}", logoutSuccess);
+                                if (event.getData().equals(session)) {
```

Comment:
can we add this in a && clause of the outter if clause

Reply:
Changed to one if statement
`if ((config.isUseKeycloakSso()) && (event.getData().equals(session)))`
rather than nested if statement.